__tests__/asset-proxy.ts

```diff
@@ -0,0 +1,75 @@
+import { http } from 'msw'
+
+import {
+  currentTestEnvironment,
+  expectIsPlaceholderResponse,
+} from './__utils__'
+
+beforeEach(() => {
+  globalThis.server.use(
+    http.get('https://whatever.org/image', () => {
+      return new Response('', {
+        headers: {
+          'content-type': 'image/png',
+          'Set-Cookie':
+            'sessionId=abc123; Expires=Wed, 09 Nov 2024 07:28:00 GMT; Path=/; Domain=whatever.org; Secure; HttpOnly; SameSite=None',
+        },
+      })
+    }),
+    http.get('https://whatever.org/notimage', () => {
+      return new Response('', {
+        headers: { 'content-type': 'application/json' },
+      })
+    }),
+  )
+})
+
+test('request to https://asset-proxy.serlo.org/src?url=* gets asset from url query parameter', async () => {
+  const env = currentTestEnvironment()
+  const response = await env.fetch({
+    subdomain: 'asset-proxy',
+    pathname: '/src?url=https://whatever.org/image',
```

Comment:
In the frontend PR I use `encodeURIComponent` on the urlParam before fetching.
Afaik that is best practice.

```suggestion
    pathname: `/src?url=${encodeURIComponent('https://whatever.org/image')}`,
```

Reply:
Thank you!